### PR TITLE
no xy check

### PR DESCRIPTION
**Pull request by mbredif**

### CODE REVIEW

File: lib/pc_schema.c [hunk 1]
```diff
@@ -335,28 +335,34 @@ void pc_schema_check_xyzm(PCSCHEMA *s)
 		char *dimname = s->dims[i]->name;
 		if ( ! dimname ) continue;
 		if ( strcasecmp(dimname, "X") == 0 ||
+			 strcasecmp(dimname, "transform.translation[0]") == 0 ||
```
Comment (elemoine) [inline lib/pc_schema.c:338]: Where does that come from?

**Review by elemoine — COMMENTED** [review]
Shouldn't we create this PR against the upstream branch?

File: lib/pc_schema.c (hunk 1 above)
Comment (mbredif) [reply lib/pc_schema.c:338]: It is the x component of the translation part of an affine transform provided by a ROS [TransformStamped](http://docs.ros.org/jade/api/geometry_msgs/html/msg/TransformStamped.html) message through our [fdw-rosbag](LI3DS/fdw-rosbag). 

I agree that it is too specific and that ultimately, either : 
- the rosbag fdw expose the translation part using x,y,z column names
- or overriding the x,y,z,m dims should be made possible in the xml schema (preferred)

**Comment by mbredif** [comment]
> Shouldn't we create this PR against the upstream branch?

If we can make it unspecific to the LI3DS use case, sure !
this implies:
- removing LI3DS specific dimension names
- correctly handling / reporting errors (nulls?) in all geometry casts whenever xdim or ydim is undefined

**Comment by elemoine** [comment]
> removing LI3DS specific dimension names

Yep, we can probably handle this in fdw-li3ds for now.

> correctly handling / reporting errors (nulls?) in all geometry casts whenever xdim or ydim is undefined

Yes. Do you want to do it? Or do you want me to do it? ;-)

**Comment by mbredif** [comment]
I am ok to let you port this li3ds specific PR upstream.

**Pull request closed without merging**